src/Time.php

```diff
@@ -6,6 +6,28 @@
 
 use DateTimeInterface;
 
+/**
+ * Freeze time.
+ *
+ * @param  callable|null  $callback
+ * @return mixed
+ */
+function freezeTime(?callable $callback = null)
```

Comment:
I am not sure whether this should be `?callback` or `?Closure`, or just don't add type-hint.

`$callback` on `travelTo` didn't have a type-hint: 
https://github.com/binotaliu/pest-plugin-laravel/blob/7cee7c73e96d6b37e68bb1df4ae2fba8e3d50496/src/Time.php#L48

Reply:
What we do on the other helpers?